commands/generate_test.go

```diff
@@ -170,9 +190,18 @@ func Test_generateCRDYAML(t *testing.T) {
 			t.Fatalf("%s failed: error while generating CRD YAML.", testcase.Name)
 		}
 
-		if generatedYAML != testcase.Output {
+		if !stringInSlice(generatedYAML, testcase.Output) {
 			t.Fatalf("%s failed: ouput is not as expected: %s", testcase.Name, generatedYAML)
 		}
 	}
 
 }
+
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
```

Comment:
I might be misreading this but aren't we likely to exit on the first item in the range?

I'd have thought we'd need to switch the logic and return false in the range iteration if `!=` and then return true at the end if we made it through the list?